template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/databricks.yml.tmpl

```diff
@@ -1,5 +1,9 @@
 # The name of the bundle. run `databricks bundle schema` to see the full bundle settings schema.
 bundle:
+  # Do not modify the below two lines, these autogenerated fields are used by the Databricks backend.
+  uuid: {{ bundle_uuid }}
+  template: mlops-stacks
```

Comment:
No need for this, the template name will  be inferred at runtime.

Reply:
Oh how will this be done? Are we storing some metadata after initialization to identify the template?